plugins/reolink/src/reolink-api.ts

```diff
@@ -201,16 +201,53 @@ export class ReolinkCameraClient {
         const url = new URL(`http://${this.host}/api.cgi`);
         const params = url.searchParams;
         params.set('cmd', 'GetDevInfo');
-        const response = await this.requestWithLogin({
+        let response = await this.requestWithLogin({
```

Comment:
why are these "let" and reused?

Use different variables for different request types. mutability on the response makes understanding this code later difficult.